src/openzaak/tests/schemas/referentielijsten.yaml

```diff
@@ -0,0 +1,1488 @@
+openapi: 3.0.0
```

Comment:
should be deleted in favour of `src/openzaak/selectielijst/tests/files/openapi.yaml`